driver_cpl/bld/build-namelist

```diff
@@ -287,14 +287,11 @@ foreach my $file (@files) {
     my @e = $xml->elements_by_name('entry');
     while ( my $e = shift @e ) {
 	my %a = $e->get_attributes();
+	my $val = $a{value};
```

Comment:
You're right I started to do this one way and then noticed that I just needed to move the DIN_LOC_ROOT line down below the next xmlvars initialization - is there a reason to do it twice or can I just remove the first one?